### PR TITLE
github: remove files on sync

### PR DESCRIPTION
**Pull request by lsf37**
Previously, file would only be added or modified, but not removed.

### CODE REVIEW

**Comment by lsf37** [comment]
Ah, actually that's not quite right yet. One minute.

**Review by Xaphiosis — APPROVED** [review]

**Comment by lsf37** [comment]
Ok, this should now be right.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -50,10 +50,15 @@ jobs:
     - uses: actions/download-artifact@v2
       with:
         name: site
+    # for removing files, we need to start fresh; this does not remove dot-files
+    - run: rm -rf *
     - run: tar -xvf site.tar
```
Comment (kent-mcleod) [inline .github/workflows/build.yml:55]: Won't this file have been deleted by the previous command?

Comment (lsf37) [reply .github/workflows/build.yml:55]: ahem, yes

Comment (lsf37) [reply .github/workflows/build.yml:55]: ok, have moved the `rm` further up

**Review by kent-mcleod — APPROVED** [review]

**Pull request merged**